### DISCUSSION TITLE
Loved the project, want to contribute

### DISCUSSION THREAD

**Discussion by Gupta-Anubhav12** [vitalik/django-ninja#703] [General]
Hey @vitalik , 
I loved the project and wanted to contribute, can I get some good first issues to start with, I've been working with Django and FastApi since quite a time and It'd be interesting to work on the project. 

Looking forward to a response.

**Comment by baseplate-admin** [comment]
There will be a large rework when pydantic hits 2.0.0.


If you want to contribute.  You can improve the docs.  There is a lot that can be done. Otherwise you can look through the issues and see if you can answer some questions :)

**Comment by baseplate-admin** [comment]
One major issue you can tackle is improving the error messages. 

I have reported it #684 and other people have reported it in #637 

And heres my initial findings https://github.com/vitalik/django-ninja/issues/637#issuecomment-1408192381

( i am out of bandwidth to debug this. I will really be grateful if you take a look into it )
